akka-stream/src/main/scala/akka/stream/scaladsl/Hub.scala

```diff
@@ -390,6 +390,11 @@ private[akka] class MergeHub[T](perProducerBufferSize: Int, drainingEnabled: Boo
  */
 object BroadcastHub {
 
+  /**
+   * INTERNAL API
+   */
+  @InternalApi private[akka] val defaultBufferSize = 256
```

Comment:
we don't use this in other places so could be `private val DefaultBufferSize = 256`

Reply:
Ups, it was my intention to leave hub untouched since I didn't actually change any logic.

Reply:
(but I'll be a good scout and make it a private constant)

Reply:
Dang, actually used in other places (Javadsl)